examples/orszag-tang/data/BOUT.inp_1

```diff
@@ -1,3 +1,6 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; WARNING! This is for BOUT++ < 4.0!
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
delete file?

Reply:
I didn't because I figured this is for comparison between different versions. But probably the physics model isn't compatible, and if you've got the old version of that you likely have the old input file too.

Delete it is!

(Also note that this is literally the only input file that uses ';' as a comment character -> another thing to go in v5 I think)